config/crd/bases/image.toolkit.fluxcd.io_imagepolicies.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
@relu can you delete the `controller-gen` binary from your local Go bin and rerun `make test` this will pull v0.4.1 and undo the above.

Reply:
Should be good now. Thanks!